cluster/gce/gci/configure-helper.sh

```diff
@@ -77,12 +77,21 @@ function setup-logrotate() {
     compress
     maxsize 10M
```

Comment:
I thought stdout/stderr was the default way fluentd picked up the container logs?